tests/robottelo/test_ssh.py

```diff
@@ -245,7 +245,7 @@ def test_execute_command_plain_output(self, settings):
 
         with ssh.get_connection() as connection:  # pylint:disable=W0212
             ret = ssh.execute_command(
-                'ls -la', connection, output_format='plain')
+                'ls -la', connection, output_format='base')
```

Comment:
The name of this function:
https://github.com/SatelliteQE/robottelo/blob/b3bb93d20080ffdbd7f626f55bd7d811fa231925/tests/robottelo/test_ssh.py#L237
should be modified to` test_execute_command_base_output`,
and also line267
https://github.com/SatelliteQE/robottelo/blob/b3bb93d20080ffdbd7f626f55bd7d811fa231925/tests/robottelo/test_ssh.py#L267 and https://github.com/SatelliteQE/robottelo/blob/b3bb93d20080ffdbd7f626f55bd7d811fa231925/tests/robottelo/test_ssh.py#L276

Actually, you can test the test_ssh.py locally by:
` pytest robottelo/tests/robottelo/test_ssh.py` so that we can pass the Travis testing :)

Reply:
OK, changes made. That test passes, but please checkout this branch and run the entire class of tests on your test infrastructure before merging.

Thank you